### DISCUSSION TITLE
Education Student verification

### DISCUSSION THREAD

**Discussion by mb-student** [community/community#19288] [GitHub Education]
Verification of student (or teacher) status for education.github.com benefits does not seem to allow for upload of documentation to prove your eligibility, only allows to take a picture. 

It would be helpful and less wasteful to allow upload of documents vs just a picture as everything is pretty much electronic anyway. Maybe I'm just blind and can't find the right page.

**Comment by Christian-GitDev** [comment]
Eu penso assim também, pois a câmera é muito ruim e não consigo deixar a impressão com boa qualidade!

**Comment by melvincwng** [comment]
Hi @mb-student, based on [Github Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack), there should be an 'Upload Image' button that allows you to upload an image of your student id/verification documents.

Can you kindly check if its there when you re-apply?

Let me know if this helps.

**Reply by melvincwng** [reply]
@mb-student, if you find this helpful, please kindly 'Mark as answer' so that others are aware this issue has been addressed.

Thanks and have a good day ahead 😄

**Comment by melvincwng** [comment]
@mb-student, if all fails, as a last resort, kindly contact [Github Education Support](https://support.github.com/contact/education).

Raise a ticket in the link above, and describe your issue.

They would usually get back to you within a few working days.

Hope this helps you!

**Comment by EddieMusick** [comment]
I just want to chime in and state that I agree there should be an option to upload a document. This process is awful; it's glitchy, depends on the quality of your camera, and constantly gets rejected by the automatic system that checks the uploaded document. I have tried with my phone, my iPad and my laptop and can't get anything accepted. I even tried creating a new account with just my .edu email address. Interestingly, the form does change (there is no drop-down to choose what type of document you are submitting) but there is still no upload button.

**Comment by davelsio** [comment]
This is a real issue. I am an MSc student at [NTNU Trondheim](https://www.ntnu.edu/) and I have been rejected several times because of this. My school does not provide a Student ID that is valid for identification (it specifically says so on our student card). The documents I have, acceptance letter, confirmation of enrollment, and current transcripts, are all digitally signed, but this is lost when taking a picture.

Like @webdevindev  has said, the process is awful and glitchy, and the rejection is done by an automated system. To make matters worse, I submitted a ticket to helpdesk with the electronically signed PDFs attached, **and it got closed the next minute by an automated system with the same generic message I got via email**.

I am now on my third try, but I don't have any hope to be accepted since I can't seem to get a human to do the review.
